src/common/tusb_fifo.c

```diff
@@ -180,6 +179,7 @@ static void _ff_push_n(tu_fifo_t* f, void const * app_buf, uint16_t n, uint16_t
         memcpy(ff_buf, app_buf, nLin_bytes);
 
         // Write data wrapped around
+        // TU_ASSERT(nWrap_bytes <= f->depth, );
```

Comment:
running test_write_overwritable2() with the pre-pr fifo will cause nWrap_bytes > f->depth (out of boundary access)